toolkit/tools/pkg/imagecustomizerlib/testdata/verity-usr-config.yaml

```diff
@@ -0,0 +1,83 @@
+storage:
+  bootType: efi
+  disks:
+  - partitionTableType: gpt
+    maxSize: 5120M
+    partitions:
+    - id: esp
+      type: esp
+      start: 1M
+      end: 9M
+
+    - id: boot
+      start: 9M
+      end: 1024M
+
+    - id: root
```

Comment:
I am thinking it might make sense to place the root partition after the usr partition, since the usr partition has a fixed size but the root partition can theoretically be expanded.

Reply:
Fixed.